server/core/src/irods_report_plugins_in_json.cpp

```diff
@@ -26,11 +30,24 @@ namespace irods {
              itr != plugin_list.end();
              ++itr)
         {
+
+            // Get filepath
+            const auto filepath = fmt::format("{}lib{}.so", plugin_home, *itr);
+
+            // Get checksum
+            char checksum[NAME_LEN];
+            int ret = chksumLocFile(filepath.c_str(), checksum, irods::SHA256_NAME.c_str());
+
+            if (ret < 0) {
+                irods::log(ret, "ChecksumLocFile Failure"); // TODO update message
```

Comment:
Don't forget about the TODO comment.

Reply:
Would something like this work better for the log message?

```C++
irods::log(ret, fmt::format("ChecksumLocFile Failure - Unable to calculate checksum for plugin: {}", *itr));
```

Reply:
or put it into square brackets so we can see empty strings...

```
for plugin [{}]"
```

Reply:
Use the new logger instead of `irods::log()`.

For example:
```c++
#include "irods/irods_logger.hpp"
using log_api = irods::experimental::log::api;
log_api::error("Failed to calculate checksum for plugin [{}].", *itr);
```